Validation/RecoTau/python/dataTypes/ValidateTausOnZTT_cff.py

```diff
@@ -5,9 +5,11 @@
 from PhysicsTools.HepMCCandAlgos.genParticles_cfi import *
 import PhysicsTools.PatAlgos.tools.helpers as helpers
 
+tauGenJetsForVal = tauGenJets.clone()
```

Comment:
@steggema I don't understand why this clone is needed since no modifications are done afterwards

Reply:
The point is that a MiniAOD era modifier changes process.tauGenJets, so it needs to have a different name and can't be the same object (i.e. if you write tauGenJetsForVal = tauGenJets it will just be a pointer). Maybe there's a better solution (... import as ... didn't work either).

Reply:
I see thanks